### PR TITLE
SCOR-193: new nodecellar with script runner

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: regression-blueprints/nodecellar-script-runner.3.2.1.m5.yaml [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+        description: >
+            Only required if using token based login on a private vcloud
+            director. This can be obtained by following the vcloud API
+            example docs. If you are unsure of how to obtain this, you will
+            need to use password based login.
+
+    vcloud_org:
+        type: string
+        description: >
+            Organization uuid
```
Comment (pacogomez) [inline regression-blueprints/nodecellar-script-runner.3.2.1.m5.yaml]: this should be the organization name, not the Org UUID

File: regression-blueprints/postgresql-autokey.3.2.1.m3.yaml [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+    vcloud_vdc:
+        type: string
+        description: >
+            Virtual data center name
+
+    vcloud_org:
+        type: string
+        description: >
+            Organization uuid
```
Comment (pacogomez) [inline regression-blueprints/postgresql-autokey.3.2.1.m3.yaml]: this should be the org name, not the org UUID

File: regression-blueprints/postgresql-keycontent.3.2.1.m5.yaml [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+    vcloud_vdc:
+        type: string
+        description: >
+            Virtual data center name
+
+    vcloud_org:
+        type: string
+        description: >
+            Organization uuid
```
Comment (pacogomez) [inline regression-blueprints/postgresql-keycontent.3.2.1.m5.yaml]: org name, not UUID

**Comment by pacogomez** [comment]
Other than org name instead of UUID, looks good to me. I haven't tested them though.

**Comment by 0lvin** [comment]
Fixed, please review

**Comment by pacogomez** [comment]
looks good to me

**Comment by 0lvin** [comment]
Tested on 1.2.1m5 score release.

**Pull request merged**